RDF-Pipeline/t/run-test.perl

```diff
@@ -94,6 +96,29 @@
   !system("/bin/echo '===================' '$tDir' '===================' >> '$tmpDiff'") || die;
   !system("echo '$tDir' > '$currentTest'") or die;
 
+  # Ensure that each pipeline.ttl has a different last-modified date,
+  # so that the framework will notice a change to pipeline.ttl when we go 
+  # from one test to another.
+  my $pipelineFile = "$tDir/setup-files/node/pipeline.ttl";
+  if (-e $pipelineFile) {
+    while (1) {
+      # no warnings;	# Avoid unused var warning of stat vars
+      my ($dev,$ino,$mode,$nlink,$uid,$gid,$rdev,$size,
+           $atime,$mtime,$ctime,$blksize,$blocks)
+               = stat($pipelineFile);
+      my $otherTestDir = $lastModified{$mtime};
+      if (!$otherTestDir) {
+	$lastModified{$mtime} = $tDir;
+	last;
+	}
+      warn "WARNING: $tDir \nhas a pipeline.ttl with the same modification time as the one \nin $otherTestDir\nThis may cause the framework to not notice a pipeline.ttl change\nwhen going from one test to another.  Attempting a fix by touching \nthe one in $tDir ...\n";
+      my $qf = quotemeta($pipelineFile);
+      !system("touch $qf") || die;
+      sleep 1; 	# In case we need to do it on another file later
+      warn " ... done touching.\n";
+      }
+    }
+
```

Comment:
Would it be possible to ensure different modified times stamps in the copy of the setup files process?  Does this need to be a completely separate step? 

If it does need to be a separate step, should this be made into a function for more clear encapsulation of this feature?